<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumbersBar</class>
 <widget class="QWidget" name="NumbersBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>147</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Number Formats</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTypeIn">
       <item>
        <property name="text">
         <string>Dec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bin</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bitsize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSizeIn">
       <item>
        <property name="text">
         <string>8 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128 bit</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Swap byte order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbOrder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Order</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Bin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbBinIn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding.png</normaloff>:/images/folding.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBin">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbBinOut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding-contract.png</normaloff>:/images/folding-contract.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDecIn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding.png</normaloff>:/images/folding.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>22</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDecOut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding-contract.png</normaloff>:/images/folding-contract.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbHexIn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding.png</normaloff>:/images/folding.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbHexOut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folding-contract.png</normaloff>:/images/folding-contract.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvBin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
